src/App.tsx

```diff
@@ -1,7 +1,34 @@
+import {useState} from 'react';
+
+import getArrivals from '../backend/fetchArrivals.ts';
 function App() {
```

Comment:
I would ususally see the imports grouped at the top with a space between them and the function